lesson_15/rhombus.py

```diff
@@ -0,0 +1,30 @@
+class Rhombus:
+    side_a: int
+    angle_a: int
+    angel_b: int
+
+    def __init__(self, side_a: int, angle_a: int):
+        self.side_a = side_a
+        self.angle_a = angle_a
+        self.angel_b = self.check_value_angel_b(angle_a)
+
+    def print_side_a(self):
```

Comment:
For such class representation it is better to used dunder python method __str__ where you can specify which attrs to be printed if we want to print an object.